code/modules/clothing/clothing.dm

```diff
@@ -64,6 +64,9 @@
 	/// A lazily initiated "food" version of the clothing for moths
 	var/obj/item/food/clothing/moth_snack
 
+	/// What should we get when we cut this piece of clothing
+	var/material = "cloth"
```

Comment:
Needs a less vague var name. 

I already converted `materials` into `custom_materials` once, I would rather it not have an equally vague replacement.

Reply:
Ok let me see what I can do, I did stumble into custom materials

Reply:
Should be done!